src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_decorator.py

```diff
@@ -4,6 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 import importlib
+from tkinter import FALSE
```

Comment:
This line breaks CI and it is never used:

https://dev.azure.com/azure-sdk/public/_build/results?buildId=1464021&view=logs&j=964b1a32-a564-5964-77b9-8017280f6c54&t=fd408678-da66-5f65-344b-e09411c080cb

```
==================================== ERRORS ====================================
_______________ ERROR collecting tests/latest/test_decorator.py ________________
ImportError while importing test module '/usr/lib64/az/lib/python3.6/site-packages/azure/cli/command_modules/acs/tests/latest/test_decorator.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib64/python3.6/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
/usr/lib64/az/lib/python3.6/site-packages/azure/cli/command_modules/acs/tests/latest/test_decorator.py:7: in <module>
    from tkinter import FALSE
E   ModuleNotFoundError: No module named 'tkinter'
```

Reply:
@ganga1980 May I ask why this unused import was added here?

Reply:
> @ganga1980 May I ask why this unused import was added here?

> @zhoxing-ms , Not sure why this got added automatically. please go-ahead and remove this unused import.